README.rst

```diff
@@ -14,7 +14,7 @@
    :target: https://travis-ci.org/sphinx-contrib/autoprogram
 
 This contrib extension, ``sphinxcontrib.autoprogram``, provides an automated
-way to document CLI programs.  It scans ``arparser.ArgumentParser`` object,
+way to document CLI programs.  It scans ``argparser.ArgumentParser`` object,
```

Comment:
This also seems a typo; it should be [`argparse`](https://docs.python.org/3/library/argparse.html).  🙄